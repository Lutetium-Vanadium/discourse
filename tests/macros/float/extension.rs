fn main() {
    let q = inquisition::questions![float { extension: todo!() }];
}
