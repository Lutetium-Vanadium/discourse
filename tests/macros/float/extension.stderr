error: option `extension` does not exist for kind `float`
 --> $DIR/extension.rs:2:45
  |
2 |     let q = inquisition::questions![float { extension: todo!() }];
  |                                             ^^^^^^^^^
